Entities/Special/CTF/FlagBase.as

```diff
@@ -106,6 +106,7 @@ void onTick(CBlob@ this)
 		{
 			this.Untag("flag missing");
 		}
+		this.Sync("flag missing", true);
```

Comment:
this is not necessary the tag is set on all clients without this line.

Reply:
I added this after testing because before this flag missing was not synced (it didn't work before sync / it worked after)